include/gdf/cffi/types.h

```diff
@@ -3,9 +3,9 @@
 typedef size_t gdf_size_type;
 typedef gdf_size_type gdf_index_type;
 typedef unsigned char gdf_valid_type;
-typedef	int64_t	gdf_date64;
-typedef	int32_t	gdf_date32;
-typedef	int32_t	gdf_category;
+typedef	long	gdf_date64;
```

Comment:
I assume these types need to be exposed to Pygdf, which is why they are in `types.h`?